xml/System.Windows.Media/TranslateTransform.xml

```diff
@@ -27,7 +27,7 @@
  ![Translate Matrix 100010dxdy1](~/add/media/translate-matrix.gif "Translate Matrix 100010dxdy1")  
 Typical 3x3 matrix for 2-D translations  
   
- **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.TranslateTransform> class provides several special features: <xref:System.Windows.Media.TranslateTransform> objects can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.TranslateTransform> class provides several special features: <xref:System.Windows.Media.TranslateTransform> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.TranslateTransform> class provides several special features: <xref:System.Windows.Media.TranslateTransform> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```